examples/playwright/src/theia-workspace.ts

```diff
@@ -46,15 +47,7 @@ export class TheiaWorkspace {
     }
 
     get path(): string {
-        let workspacePath = this.workspacePath;
-        if (!OSUtil.osStartsWithFileSeparator(this.workspacePath)) {
-            workspacePath = `${OSUtil.fileSeparator}${workspacePath}`;
-        }
-        if (OSUtil.isWindows) {
-            // Drive letters in windows paths have to be lower case
-            workspacePath = workspacePath.replace(/.:/, matchedChar => matchedChar.toLowerCase());
-        }
-        return workspacePath;
+        return path.normalize(this.workspacePath);
```

Comment:
I'm not sure this is equivalent to the old code: for starters, it would not prepend a `/` before a relative path on unix, like the old code did.

Reply:
Right, they do not behave the same. I also wondered what the original code was trying to solve and assumed it was supposed to deal with cross-platform paths, especially on Windows, and drive letters.

I also think, that the `/` before relative paths on Unix systems was an unintentional side effect in the old code. If you prefer the original version to `path.normalize()`, I'll of course revert it back.

Reply:
Right, `path.normalize()` does not make drive letters lowercase, which seems to be at least necessary when running playwright from vscode. I reverted my change.